### ISSUE TITLE
Cannot get the user who run a pipeline

### ISSUE THREAD

**Issue by martincandal** [openshift/jenkins-sync-plugin#366]
I cannot find a way to get the user that trigger the pipeline in OpenShift from Jenkins. If I go to the console log of the executed job, it not shows the user that run it.
I am not using OAuth authentication. I have integrated OpenShift and Jenkins to Active Directory.

Thanks!

**Comment by waveywaves** [comment]
@martincandal Could you send the logs over so we could check it out ?
Apart from that also explain more on how you have architected you Jenkins infra. Seems that the jobs are not being executed through the sync plugin from what you are saying.

**Comment by martincandal** [comment]
Hi,
We recently moved out from the Jenkins that RedHat provides to Cloudbees Core, an Enterprise Jenkins.
We successfully configured the OpenShift Sync Plugin and OpenShift Client and all is working correctly.
Now, we want to take some metrics about the process, like the result of building an application, who performed the approval, etc. But we cannot find what users triggers a build. This is important for us, because we need to keep that traking of everithing we do. 
I don't know what log sent you, but when we run a job directly from Cloudbees, we see this line:
Started by user Martin N Candal

If we run the same build fom openshift, this information is not available.
Thanks!
